templates/web/base/waste/garden/subscribe_confirm.html

```diff
@@ -29,7 +29,11 @@ <h1 class="govuk-panel__title">
   [% ELSIF action == 'add_containers' %]
     Bins typically arrive within two weeks, but this may vary due to demand.
   [% ELSE %]
+    [%- IF c.cobrand.moniker == 'brent' -%]
+    Changes to your subscription will show up within 24 hours.
```

Comment:
Note as I mentioned on BC, this was only shown to staff, as normal users would get the message a bit above but no harm in changing anyway, thanks.